src/main/java/org/jabref/gui/JabRefFrame.java

```diff
@@ -278,6 +278,7 @@ private void init() {
          */
         EasyBind.subscribe(tabbedPane.getSelectionModel().selectedItemProperty(), e -> {
             if (e == null) {
+                Globals.stateManager.activeDatabaseProperty().setValue(Optional.empty());
```

Comment:
Hm, Optional sollte man eigentlich nicht als Parameter nutzen.

Reply:
Yes, but this is the only way because `activeDatabaseProperty` uses the generic `Property<T>`, which provides `setValue(T value)`.